src/NativeEventEmitter.re

```diff
@@ -0,0 +1,17 @@
+type t;
+
+type emitterSubscription;
+
+external create : NativeModules.t => t =
+  "NativeEventEmitter" [@@bs.new] [@@bs.module "react-native"];
+
+external addListener : t => string => (Js.Dict.t 'a => unit) => emitterSubscription = "" [@@bs.send];
```

Comment:
Well, if you got something working with the API then we got with your version :)